### ISSUE TITLE
Does not build on OS X

### ISSUE THREAD

**Issue by mattstratton** [mh-cbon/gh-api-cli#7]
I'm trying to build this on OS X, as Mac binaries are not provided...I get the following error:

```
~/go/src/github.com/mh-cbon/gh-api-cli  master 4.0.3  go get ./...                                                                                                                                                                                                                                                                                                                                              2 ↵  2255  00:37:17
# github.com/mh-cbon/gh-api-cli/gh
gh/gh.go:58: not enough arguments in call to client.Authorizations.Create
	have (*github.AuthorizationRequest)
	want ("context".Context, *github.AuthorizationRequest)
gh/gh.go:69: not enough arguments in call to client.Authorizations.List
	have (*github.ListOptions)
	want ("context".Context, *github.ListOptions)
gh/gh.go:90: not enough arguments in call to client.Authorizations.Delete
	have (int)
	want ("context".Context, int)
gh/gh.go:142: not enough arguments in call to client.Repositories.ListReleases
	have (string, string, *github.ListOptions)
	want ("context".Context, string, string, *github.ListOptions)
gh/gh.go:169: not enough arguments in call to client.Repositories.ListReleaseAssets
	have (string, string, int, *github.ListOptions)
	want ("context".Context, string, string, int, *github.ListOptions)
gh/gh.go:237: not enough arguments in call to client.Users.Get
	have (string)
	want ("context".Context, string)
gh/gh.go:250: not enough arguments in call to client.Repositories.CreateRelease
	have (string, string, *github.RepositoryRelease)
	want ("context".Context, string, string, *github.RepositoryRelease)
gh/gh.go:262: not enough arguments in call to client.Repositories.DeleteRelease
	have (string, string, int)
	want ("context".Context, string, string, int)
gh/gh.go:270: not enough arguments in call to client.Repositories.DeleteReleaseAsset
	have (string, string, int)
	want ("context".Context, string, string, int)
gh/gh.go:319: not enough arguments in call to client.Repositories.UploadReleaseAsset
	have (string, string, int, *github.UploadOptions, *os.File)
	want ("context".Context, string, string, int, *github.UploadOptions, *os.File)
gh/gh.go:319: too many errors
```

**Comment by mh-cbon** [comment]
you need this commit of go-github https://github.com/google/go-github/commit/90861904aa86352a956f91ba0664df93a3547584

one should fix the glide to include that hash to the dependencies. Not sure how it found the right commit at https://travis-ci.org/mh-cbon/gh-api-cli/jobs/273003130

**Comment by mh-cbon** [comment]
check also https://github.com/mh-cbon/gh-api-cli#install

**Comment by mh-cbon** [comment]
fixed in 5.0.0

**Comment by mh-cbon** [comment]
added mac binaries at https://github.com/mh-cbon/gh-api-cli/releases/tag/5.0.0
